Assets/Python/GreatPeople.py

```diff
@@ -606,59 +696,146 @@ def setup():
 },
 iCivPersia : {
 	iGreatProphet : [
-		"Zoroaster", # 18-10th BC
+		"Mahabad", # legendary
+		"Zarathustra", # 18-10th BC
 		iClassical,
+		"Baga-data", # 3rd BC
 		"Mani", # 3rd
+		"Kartir", # 3rd
 		"Mazdak", # 4th
+		"Mar Aba", # 6th
 		iMedieval,
+		"Al-Muqanna", # 8th
 		"Al-Ghazali", # 11th
 		"Mevlana", # 13th
 		iRenaissance,
 		"Mulla Sadra", # 17th
+		"Muhammad Baqir Majlisi", # 17th
+		"Muhammad Baqir Behbahani", # 18th
+		iIndustrial,
+		u"Bahá'u'lláh", # 19th
+		u"Báb", # 19th
+		iGlobal,
+		"Muhammad Husayn Tabataba'i", # 20th
+		"Muhammad-Taqi Mesbah-Yazdi", # 20th
 	],
 	iGreatArtist : [
-		"Firdausi", # 10th
+		"Pahlbod", # 7th
+		"fNagisa", # 7th
+		"Bamshad", # 7th
+		iMedieval,
+		"Rudaki", # 9th
+		"Ferdowsi", # 10th
 		"fRabia Balkhi", # 10th
+		"Nizami Ganjavi", # 12th
 		"Safi al-Din", # 13th
 		"Saadi", # 13th
+		"Nur ad-Din Abd ar-Rahman Jami", # 15th
 		iRenaissance,
 		"Kamal ud-Din Behzad", # 15th
-		"Reza Abbasi", # 16th
+		"Reza Abbasi", # 17th
+		"Mesrop", # 17th
+		iIndustrial,
+		"Mihr 'Ali", # 19th
+		iGlobal,
+		"fForough Farrokhzad", # 20th
+		"Hossein Amanat", # 20th
+		"fHayedeh", # 20th
 	],
 	iGreatScientist : [
+		"Ktesias", # 5th BC
 		"Ardashir", # 4th
+		"Borzuya", # 6th
+		"Paulos-e irani", # 6th
 		iMedieval,
-		"Al-Khwarizmi", # 9th
-		"Al-Razi", # 9th
+		"Muhammad ibn Musa al-Khwarizmi", # 9th
+		"Muhammad ibn Zakariya al-Razi", # 9th
 		"Ibn Sina", # 10th
 		"Abd al-Rahman al-Sufi", # 10th
-		"Al-Farisi", # 13th
+		"Kamal al-Din al-Farisi", # 13th
+		iRenaissance,
+		"Al-Birjandi", # 16th
+		"Baha al-din al-Amili", # 16th
+		"Qazi Sa'id Qumi", # 17th
+		"Alavi Shirazi", # 17th
+		iGlobal,
+		"Mahmoud Hessaby", # 20th
+		"Ali Javan", # 20th
+		"Cumrun Vafa", # 20th
 	],
 	iGreatMerchant : [
+		"Kroisos", # 6th BC
+		"Athurpat", # 4th BC
 		"Kavadh", # 5th
 		iMedieval,
 		"Ahmad ibn Rustah", # 10th
 		"Istakhri", # 10th
+		iGlobal,
+		"Manny Mashouf", # 20th
+		"Nasser David Khalili", # 20th
+		"Omid Kordestani", # 20th
+		"Amir Ansari", # 20th
+		"Pierre Omidyar", # 20th
 	],
 	iGreatEngineer : [
-		"Artaxerxes", # 5th BC
+		u"Artaxsaça", # 4th BC
```

Comment:
I would prefer the more phonetic transliteration "Artakhshathra" (see [here](http://www.jewishencyclopedia.com/articles/1827-artaxerxes-i))